python/torch_mlir/extras/fx_importer.py

```diff
@@ -220,19 +220,35 @@
     "gt": torch.ops.aten.gt,
 }
 
-SYMBOLIC_TORCH_OPS = {
-    torch.ops.aten.sym_size,
-    torch.ops.aten.sym_stride,
-    torch.ops.aten.sym_numel,
-}
-
-SYMBOLIC_OP_TO_TORCH_OP = {
-    (torch.ops.aten.sym_size, 1): torch.ops.aten.size.default,
-    (torch.ops.aten.sym_size, 2): torch.ops.aten.size.int,
-    (torch.ops.aten.sym_stride, 1): torch.ops.aten.stride.default,
-    (torch.ops.aten.sym_stride, 2): torch.ops.aten.stride.int,
-    (torch.ops.aten.sym_numel, 1): torch.ops.aten.numel.default,
-}
+# this allows stripping of cuda versions like: "+cu113"
```

Comment:
I don't understand what this comment means?

Reply:
when torch installs with cuda,  torch.__version__ = "2.1.0+cu113". In order to support both "2.1.0" and "2.1.0+cu*" split at the plus and then compare. 

"2.1.0+cu113" <= "2.1.0" resolves to False. Maybe there is a fancier way to do it but 🤷